<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkProperties</class>
 <widget class="QWidget" name="MarkProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mark Plugin</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/static/default/icon/22x22/edit-select.png</normaloff>:/static/default/icon/22x22/edit-select.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Radius</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Line color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="topMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <widget class="QFrame" name="lineColorFrame">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>14</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="newMarkButton">
        <property name="text">
         <string>New</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../measure/resources/ui/qt/resources.qrc">
          <normaloff>:/static/default/icon/22x22/document-new.png</normaloff>:/static/default/icon/22x22/document-new.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="deleteMarkButton">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../measure/resources/ui/qt/resources.qrc">
          <normaloff>:/static/default/icon/22x22/edit-delete.png</normaloff>:/static/default/icon/22x22/edit-delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="thicknessBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="lock">
        <property name="text">
         <string>Lock</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="visible">
        <property name="text">
         <string>Visible</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="markGroup">
     <property name="title">
      <string>Marks</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="../../../../measure/resources/ui/qt/resources.qrc"/>
 </resources>
 <connections/>
</ui>
